src/run.ps1

```diff
@@ -29,6 +29,24 @@ $AzExtensions=@{
     "k8s-extension"="";
     "customlocation"=""};
 
+$logFile = Join-Path $PSScriptRoot "arcavs-output.log"
```

Comment:
What is this new log file for?
Do we need this file for debugging purposes now? If yes, we should update our TSGs

Reply:
Sure, let's update the TSG. This log file is focused on the az installation logs.